autoload/lsp/ui/vim/completion.vim

```diff
@@ -191,10 +191,7 @@ function! s:clear_inserted_text(line, position, completed_item, completion_item)
 
   " Expand remove range to textEdit.
   if has_key(a:completion_item, 'textEdit')
-    let l:range['start']['character'] = min([
-          \   l:range['start']['character'],
-          \   a:completion_item['textEdit']['range']['start']['character']
-          \ ])
+    let l:range['start']['character'] = a:completion_item['textEdit']['range']['start']['character']
```

Comment:
I discussed with mattn. It is OK.
I checked with `vim-language-server`, `intelephense`, `html-languageserver` works all.

Reply:
Thank you! I checked metals, julia-language-server, css-languageserver, clangd.